src/Game/MultiplayerMode.cpp

```diff
@@ -36,8 +30,17 @@ MultiplayerMode::DoMove(uint8_t x,
   // Сделаем направление ходов и если true, то вычесть очки
   if (!IGame::DoMove(x, y, directions))
     return false;
+
   // Добавим очки к активному игроку
   return m_list_player[m_active_user].AddPoints((int16_t)directions.size());
+
+  // Если достигли последнего игрока, то поставим первого
+  if (m_active_user == (m_list_player.size() - 1))
+    m_active_user = 0;
+  else
+    // Переключим пользователя
+    ++m_active_user;
+
```

Comment:
Зачем так сложно? Вот:
```suggestion
  m_active_user = (m_active_user + 1) % m_list_player.size();
```